crates/cdk/src/generate.rs

```diff
@@ -17,9 +20,17 @@ static CONNECTOR_TEMPLATE: Dir<'static> =
 /// Generate new SmartConnector project
 #[derive(Debug, Parser)]
 pub struct GenerateCmd {
-    /// SmartConnector Project Name
+    /// Connector Name
     name: Option<String>,
 
+    #[arg(long, value_name = "GROUP")]
+    /// Connector developer group
+    group: Option<String>,
```

Comment:
Isn't this already available in the connector metadata? why not just pass connector config rater than duplicate here

Reply:
This would be the value used on generation to set the `group` field in `Connector.toml`. This is equivalent to replying the prompt on:

```
Please set a group name
```

In the following output:

```
cdk generate
🤷   Project Name: test
🤷   Please set a group name: infinyon
✔ 🤷   Which type of Connector would you like [source/sink]? · source
✔ 🤷   Will your Connector be public? · true
[1/8]   Done: .gitignore
[2/8]   Done: Cargo.toml
[3/8]   Done: Connector.toml
[4/8]   Done: README.md
[5/8]   Done: sample-config.yaml                                                [6/8]   Done: src/config.rs                                                     [7/8]   Done: src/main.rs                                                       [8/8]   Done: src
```

Reply:
Got it